modules/clients.py

```diff
@@ -27,51 +27,43 @@ def done_at(self, time):
 
 class Clients(object):
 
-    def __init__(self, env, server, requests, process_time=0.00001, create_request_rate=0.01, max_requests=float("inf")):
+    def __init__(self, env, server, requests, sleep_time=0.00001, create_request_time=0.01, max_requests=float("inf"), 
```

Comment:
Is this properly formatted?

Reply:
> Is this properly formatted?
Sorry, I didn't understand the question.
By the way, we are planning start these attributes using a config file. So, a lot of those parameters will be removed.

Reply:
Sorry, I didn't understand the question.
By the way, we are planning start these attributes using a config file. So, a lot of those parameters will be removed.